QUICKSTART.md

```diff
@@ -0,0 +1,351 @@
+
+to apply it. Use
+
+  KUBECONFIG=<edge-cluster-config> kubectl get deployment -n "kcp-edge-syncer-florin-5c4r0a44" kcp-edge-syncer-florin-5c4r0a44
+
+to verify the syncer pod is running.
+```
+
+An edge syncer manifest yaml file is created in your current director: `florin-syncer.yaml`. The default for the output file is the name of the SyncTarget object with “-syncer.yaml” appended.
+
+Now deploy the edge syncer to the `florin` edge cluster:
+
+  
+```shell
+KUBECONFIG=$florin_kubeconfig kubectl apply -f florin-syncer.yaml
+```
+
+which should yield something like:
+
+```console
+namespace/kcp-edge-syncer-florin-5c4r0a44 created
+serviceaccount/kcp-edge-syncer-florin-5c4r0a44 created
+secret/kcp-edge-syncer-florin-5c4r0a44-token created
+clusterrole.rbac.authorization.k8s.io/kcp-edge-syncer-florin-5c4r0a44 created
+clusterrolebinding.rbac.authorization.k8s.io/kcp-edge-syncer-florin-5c4r0a44 created
+role.rbac.authorization.k8s.io/kcp-edge-dns-florin-5c4r0a44 created
+rolebinding.rbac.authorization.k8s.io/kcp-edge-dns-florin-5c4r0a44 created
+secret/kcp-edge-syncer-florin-5c4r0a44 created
+deployment.apps/kcp-edge-syncer-florin-5c4r0a44 created
+```
+
+Check that the edge syncer pod is running:
+
+```console
+KUBECONFIG=$florin_kubeconfig kubectl get pods -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE                         NAME                                              READY   STATUS    RESTARTS   AGE
+kcp-edge-syncer-florin-5c4r0a44   kcp-edge-syncer-florin-5c4r0a44-bb8c8db4b-ng8sz   1/1     Running   0          30s
+kube-system                       coredns-565d847f94-kr2pw                          1/1     Running   0          85s
+kube-system                       coredns-565d847f94-rj4s8                          1/1     Running   0          85s
+kube-system                       etcd-florin-control-plane                         1/1     Running   0          99s
+kube-system                       kindnet-l26qt                                     1/1     Running   0          85s
+kube-system                       kube-apiserver-florin-control-plane               1/1     Running   0          100s
+kube-system                       kube-controller-manager-florin-control-plane      1/1     Running   0          100s
+kube-system                       kube-proxy-qzhx6                                  1/1     Running   0          85s
+kube-system                       kube-scheduler-florin-control-plane               1/1     Running   0          99s
+local-path-storage                local-path-provisioner-684f458cdd-75wv8           1/1     Running   0          85s
+``` 
+
+### c. Create the nginx workload and deploy it to the florin cluster
+
+Create the `EdgePlacement` object for your workload. Its “where predicate” (the locationSelectors array) has one label selector that matches the Location object (`florin`) created earlier, thus directing the workload to `florin` edge cluster.
```

Comment:
I would switch the order around, to match the fact that the EdgePlacement logically depends on the workload description and not the other way around.